pages/community/docs/getting-started/become-a-tsc-member.md

```diff
@@ -0,0 +1,103 @@
+
+## What is the Technical Steering Committee?
+TSC is a group of maintainers that contribute to the creation and maintenance of AsyncAPI Initiative.
+
+## Why the TSC was formed
+The concept of TSC was formed when AsyncAPI joined the Linux Foundation in 2021 to guarantee a system of open governance that promotes ownership and neutrality of all participants in the initiative. 
+
+
+ 
+Some of the principles of the Linux Foundation partnership include:
+
+* No founding members with special privileges
+* Equality among all contributors
+* Asynchronous decision making 
+* Every maintainer is allowed to automatically become a TSC member
+
+## Benefits of becoming a TSC member
+
+As a TSC member you will enjoy the following benefits: 
+ 
+* Recognition from AsyncAPI community
+* Exposure of your portfolio to potential employers
+
+* Involved in the voting process on topics concerning the entire Initiative
+* Included in key decision-making for the Initiative. Especially related to finance. 
+
+
+## How to become a TSC member
+Any maintainer can be a TSC member. In other words, to become a TSC member, you need to become a maintainer in one of the projects in the AsyncAPI Initiative. 
+
+You can do it by doing at least one of below things in AsyncAPI Initiative GitHub organization:
+* Initiate a new repository/project and become its maintainer
+* Donate existing repository/project and stay as its maintainer
+* Support and commit to existing projects and be invited by other maintainers to join them as a maintainer
+
+## Make your first contribution
+We know you are nervous to make your first contribution. In this section we guide you on steps to becoming a TSC member:
+
+1. Pick an issue
+2. Open a pull request
+3. Get it merged
+4. Get invitation to be a code owner
+5. Become a TSC member
+
+**Step 1: Pick an Issue**
+
+Here's how to select an issue to contribute to:
+* Join existing PR reviews
+* Look for issues with [**good first issue** label](https://github.com/issues?page=1&q=is%3Aopen+org%3Aasyncapi+sort%3Aupdated-desc+label%3A%22good+first+issue%22)
+* Join [AsyncAPI Slack community](https://asyncapi.com/slack-invite) and join the `#11_how-to-contribute` channel
+* Ask maintainers to do a public live stream called **Let's talk about contributing** where they can explain specific part of the project where you want to contribute
+
+**Step 2: Open a pull request**
+
+To open a pull request successfully, follow our [Git workflows](https://github.com/asyncapi/community/blob/master/git-workflow.md) contributors guide.
+
+Use the following steps to open your pull request:
+1. Configure a fork
+2. Work on it locally 
+3. Write an appropriate PR title using [conventional commits specification](https://github.com/asyncapi/.github/blob/master/CONTRIBUTING.md#conventional-commits)
+4. Test PR against Github checks-ensure they all pass
+
+**Step 3: Get it merged**
```

Comment:
```suggestion
**Step 3: Get pull request merged**
```

Always avoid using antecedents without defining them clearly in each sentence 😄✌🏽